### PR TITLE
chore(version): update version number & publisher info

### PR DESCRIPTION
**Pull request by andy-blum**
Mimics 9ea9d66 to trigger new publish & updates publisher information to remove @photodow

Worth noting that this package hasn't been updated since Jan 19, 2022 - I'm not sure what kinds of gremlins we may run into with this release, but the chrome dashboard has a "rollback" button.

##### Changed

- changes package name to `carbon-devtools-v10`
- changes publisher to `IBM`
- changes email address to `carbon@us.ibm.com`

### CODE REVIEW

File: packages/web-extension/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "carbon-devtools",
-  "version": "2.7.7",
+  "name": "carbon-devtools-v10",
```
Comment (mattrosno) [inline packages/web-extension/package.json]: Do you mean to set the extension's display name as well? Or does this string get built into the display name? 
image

Comment (andy-blum) [reply packages/web-extension/package.json]: I didn't see anywhere to modify a "display name". I'm wondering if there's not a way to change the extension name.

Comment (mattrosno) [reply packages/web-extension/package.json]: Per https://developer.chrome.com/docs/extensions/reference/manifest/name, the name is set in the manifest.

https://github.com/carbon-design-system/devtools/blob/base/packages/web-extension/src/manifest.json#L2

That name is set during the webpack build.

https://github.com/carbon-design-system/devtools/blob/base/packages/web-extension/webpack.common.js#L166

I'm not suggesting that the name should be hardcoded or set build-time. Just sharing where it's happening.

Comment (andy-blum) [reply packages/web-extension/package.json]: Thanks for the pointer on this, I've just confirmed that building the package (`yarn build`) does alter the manifest.json file's name. Updates incoming.

**Comment by andy-blum** [comment]
Screenshot 2024-07-22 at 2 17 36 PM

**Review by mattrosno — APPROVED** [review]
LGTM! 🎉

**Pull request merged**